### DISCUSSION TITLE
PyVista sprint in 2022 🚀

### DISCUSSION THREAD

**Discussion by tkoyama010** [pyvista/pyvista#2995] [Announcements]
Welcome to PyVista sprint in 2022 🚀

You can contribute PyVista project to solve the [good first issue](https://github.com/pyvista/pyvista/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22). 
- If you find the issue you want to solve please comment here.
- If you do not find the issue you can solve please mention @tkoyama010 . 
- Feel free to join PyVista [slack](https://join.slack.com/t/pyvista/shared_invite/zt-1cwhtdgo0-4_RKBZT8Uow9KIQUWhqOpg) or [Discord](https://github.com/pyvista/pyvista/discussions/2749) !

You can see our contribution in [label scipy2022](https://github.com/pyvista/pyvista/pulls?q=label%3Ascipy2022+).

**Comment by ENIAC-6** [comment]
Hello

**Reply by tkoyama010** [reply]
Hi @ENIAC-6 . Could you solve #2599 ?

**Reply by tkoyama010** [reply]
Thanks for solving it. Could you solve #2512 next?

**Reply by RobPasMue** [reply]
I am on that one @tkoyama010 :)

**Reply by tkoyama010** [reply]
Sorry to duplicate we are happy to merge both :)

**Reply by tkoyama010** [reply]
@ENIAC-6 I recommend you to do #1629.

**Comment by meverson86** [comment]
Hello everyone!  I'm not officially part of the sprint, but would like to try to knock out #2529 while I'm waiting for flights.  Have already started putting thoughts down on the issue.

**Reply by tkoyama010** [reply]
Nice to meet you @meverson86 . Thanks for the comment. Welcome to contribution!

**Comment by RobPasMue** [comment]
Hi there! I'd like to participate as well on the sprint! Nice to meet you all. Happy to help in any existing issue.

**Reply by tkoyama010** [reply]
Thanks @RobPasMue . Could you solve #2512 ?

**Reply by RobPasMue** [reply]
PR ready for review #2999

**Reply by RobPasMue** [reply]
Can I tackle #2769?

**Reply by tkoyama010** [reply]
Of course.

**Reply by RobPasMue** [reply]
PR ready for review #3001

**Reply by RobPasMue** [reply]
Looking into #1625, but I do not see the problem. Maybe some insight? @tkoyama010

**Reply by RobPasMue** [reply]
I'll leave #1625 on stand-by for now. Waiting confirmation from @akaszynski. Working on #1580 in the meantime.

**Reply by RobPasMue** [reply]
Working on #1625 for the last day of the sprint @tkoyama010

**Reply by RobPasMue** [reply]
PR #3011 ready for review! It will solve #1625

**Reply by tkoyama010** [reply]
@RobPasMue Great contribution!

**Comment by tkoyama010** [comment]
@RobPasMue @ENIAC-6 Feel free to find a [good first issue](https://github.com/pyvista/pyvista/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22). And we don't care about the duplicate. Making pull requests and reviewing each other is also essential.

**Reply by ENIAC-6** [reply]
I'll take #2913.

**Reply by RobPasMue** [reply]
I'm on #2769

**Comment by tkoyama010** [comment]
@ENIAC-6 You can use [this](https://docs.pyvista.org/api/plotting/_autosummary/pyvista.Plotter.add_legend.html) for `remove_legend` example.

**Reply by tkoyama010** [reply]
https://stackoverflow.com/questions/5893163/what-is-the-purpose-of-the-single-underscore-variable-in-python

**Reply by tkoyama010** [reply]
https://github.com/pyvista/pyvista/issues/1629

**Reply by tkoyama010** [reply]
https://github.com/pyvista/pyvista/compare/main...ENIAC-6:pyvista:patch-4

**Comment by charithgunasekara** [comment]
hi

**Reply by tkoyama010** [reply]
Hi. Welcome to the PyVista contribution! Could you add an example section to `link_views` method's docstring?
https://github.com/pyvista/pyvista/blob/db42e74a65e092f7689c0a5323a1fca538623766/pyvista/plotting/plotting.py#L3130-L3140

**Reply by tkoyama010** [reply]
To see a docstring format, you can use this link.
https://numpydoc.readthedocs.io/en/latest/format.html#examples
And one of the examples which is used `link_views` is here.
https://docs.pyvista.org/examples/01-filter/gradients.html
